shared/stats/rollup.config.mjs

```diff
@@ -39,24 +50,35 @@ export default defineConfig(
 
           // Stats: File size
           visualizer({
-            filename: join(
-              'dist',
-              dirname(modulePath),
-              `${parse(modulePath).name}.yaml`
-            ),
+            emitFile: true,
+            filename: `${parse(modulePath).name}.yaml`,
+            projectRoot: dirname(packagePath),
+            template: 'list',
+
+            // Skip sourcemaps to use original file sizes
+            sourcemap: false
+          }),
+
+          // Stats: File size (minified)
+          visualizer({
+            emitFile: true,
+            filename: `${parse(modulePath).name}.min.yaml`,
             projectRoot: dirname(packagePath),
-            template: 'list'
+            template: 'list',
+
+            // Use sourcemaps to calculate minified sizes
+            sourcemap: true
           }),
 
           // Stats: Module tree map
           visualizer({
-            filename: join(
-              'dist',
-              dirname(modulePath),
-              `${parse(modulePath).name}.html`
-            ),
+            emitFile: true,
+            filename: `${parse(modulePath).name}.html`,
             projectRoot: dirname(packagePath),
-            template: 'treemap'
+            template: 'treemap',
+
+            // Use sourcemaps to calculate minified sizes
+            sourcemap: true
```

Comment:
I'm not opposed to this at all, but good to get your rationale for using minified sizes for the tree map.

Reply:
We use Babel to [preserve JSDoc blocks with `@param`, `@returns` and `@typdef`](https://github.com/alphagov/govuk-frontend/blob/f8aa28e2c34fc6803c59f124747d675f5568a9ac/packages/govuk-frontend/babel.config.js#L29-L32) (unless private/internal)

Browsing the map points out "which module is most documented" rather than the largest, so by defaulting to the minified size it better represents what ends up in browsers

Gzip/Brotli next one day?